packages/components/src/Subscription/Component.purs

```diff
@@ -160,7 +163,11 @@ render self@{ props: props@{ subscription: sub@{ package } } } =
         where
           asyncWrapper = AsyncWrapper.asyncWrapper
             { wrapperState: self.state.wrapperProgress
-            , readyView: actionsContainer [ pauseIcon, temporaryAddressChangeIcon, deliveryReclamationIcon ]
+            , readyView: actionsContainer $ [ pauseIcon ]
```

Comment:
We don't need to wrap these in arrays, I think it would be cleaner/easier to read as
```purescript
[ pauseIcon
, if maybe true Array.null self.state.pausedSubscriptions
  then mempty
  else removeSubscriptionPauses
, temporaryAddressChangeIcon
, deliveryReclamationIcon
]
```